gateway/main.go

```diff
@@ -26,15 +25,9 @@ func (g *Gateway) authorize(ctx context.Context, header http.Header) error {
 	}
 	header.Add("authorization", fmt.Sprintf("Bearer %s", user.Token))
 	header.Add("x-source", CLI_SOURCE_HEADER)
-	return nil
-}
-
-func (g *Gateway) setProjectToken(ctx context.Context, req *gql.Request) error {
 	if g.cfg.RailwayProductionToken == "" {
-		return errors.ProductionTokenNotSet
+		header.Add("project-access-token", g.cfg.RailwayProductionToken)
 	}
```

Comment:
This adds the token to the header if it empty. I think it should be the other way around

Reply:
Wow, odd that it worked then... I suppose it just used my creds will test with a deployment I'm not linked' to